bin/serve_update_description_gen/go.mod

```diff
@@ -0,0 +1,15 @@
+module serve_update_description_gen
```

Comment:
It could be extended as sub-commands in one cli, just a thought (docker cli as an example)

Reply:
but, in this way, it will broke compatibility anyway, or i should add call for gen function now... hm, sounds intresting anyway, but force struct change) cmd/updaterini should keep cli app, functions should be in a different packages

Reply:
let's wait for another CMD, without it this structure seems to be confusing

Reply:
But, you are reviewer, do as you say)

Reply:
rework it now